Automattic/pocket-casts-android
# Create Kids Dialog

## Description
- Creates the following forms for kids feature when tapping on `Request Early Access`
- See: G60UqxPhTynbalo0Vfr1NC-fi-1853_16704

I will work on the transition between the two new dialogs in a later PR 

Fixes #2510
Fixes #2509

## Testing Instructions
1. Have KIDS_PROFILE feature flag enabled
2. Go to profile
3. Tap on Request Early access
4. You should see the `Thank you for your interest` bottom sheet dialog
5. Tap on `No, thank you` button
6. The dialog should be dismissed
7. Tap on Request Early access again
8. Tap on `Send feedback` button
9. The next dialog with a form should be opened


## Screenshots or Screencast 
[Screen_recording_20240731_103005.webm](https://github.com/user-attachments/assets/0aeed71f-b267-440b-bc96-66e6360f5625)


<img width="1620" alt="image" src="https://github.com/user-attachments/assets/ed27a5f0-2ea6-44be-bee9-6d4a72ba4e21">

<img width="865" alt="image" src="https://github.com/user-attachments/assets/d4de8433-37bd-47dc-9641-e6ddcf09f166">


## Checklist
 
#### I have tested any UI changes...
